eng/pipelines/client.yml

```diff
@@ -13,8 +18,7 @@ pr:
 trigger:
 - master
 
-variables:
-  ProjectFile: eng/data-plane.proj
```

Comment:
Any particular reason that you removed this variable?

Reply:
We have to remove the variable otherwise we cannot set it at queue time.